charts/hono/templates/_helpers.tpl

```diff
@@ -102,7 +102,7 @@ The scope passed in is expected to be a dict with keys
 - "component": the value to use for the "app.kubernetes.io/component" label
 */}}
 {{- define "hono.metadata" -}}
-name: {{ printf "%s-%s" .dot.Release.Name .name | quote }}
+name: {{ printf "%s-%s" (include "hono.fullname" .dot ) .name | quote }}
```

Comment:
It looks like there are still some more replacements like this needed:
- the pod list in `NOTES.txt`
- data-grid server name in `hono-data-grid.xml`
- host entries in `qdrouterd.json`
- `HTTP_BASE_URL` in `add_example_data_device_registry.sh`
- `{{ .Release.Name }}-kafka-example-keys` in `values.yaml`

Reply:
done